apps/docs/components/ui/sectionPopover/sectionPopover.css

```diff
@@ -0,0 +1,76 @@
+.hd-section-popover__button {
+    display: flex;
+    cursor: pointer;
+    background-color: transparent;
+    align-items: center;
+    border: none;
+    gap: var(--hd-space-05);
+    color: var(--hd-color-neutral-text-weak);
+    padding-inline-start: var(--hd-space-2);
+    padding-inline-end: var(--hd-space-1);
+    padding-block: var(--hd-space-05);
+}
+
+.hd-section-popover__button-icon {
+    stroke: var(--hd-color-neutral-icon);
+    transition: rotate .1s ease-in;
+    rotate: -90deg;
+}
+
+.hd-section-popover__button--open svg {
+    rotate: 0deg;
+}
+
+@media screen and (width >= 48rem) {
+    .hd-section-popover__button {
+        display: none;
+    }
+}
+
+.hd-section-popover__button:hover {
+    color: var(--hd-color-neutral-text-weak-hover);
+}
+
+/* Povover */
+.hd-section-popover__popover {
+    position: absolute;
+    left:var(--hd-space-3)!important;
+    top: var(--hd-space-9);
+    width: calc(100% - var(--hd-space-6));
+    border: var(--hd-border-size) solid var(--hd-color-neutral-border);
+    background-color: var(--hd-color-neutral-surface);
+    border-radius: var(--hd-space-1);
+    padding: var(--hd-space-1) var(--hd-space-3) var(--hd-space-2) var(--hd-space-3);
+    box-shadow: 0 0.75rem 2rem rgba(0 0 0 / 10%), 0 0.125rem 0.375rem rgba(0 0 0 / 8%);
+    display: none;
+}
+
+.hd-section-popover__popover--open {
+    display: block;
+}
+
+/* list */
+.hd-section-popover__list {
+    list-style-type: none;
+}
+
+.hd-section-popover__list-link {
+    color: var(--hd-color-neutral-text-weakest);
+    line-height: 1.875rem;
+    display: flex;
+    font-size: 0.875rem;
```

Comment:
this should be a css variable

Reply:
There is no determined font-size scale as of now.

Reply:
It's a possibility, currently the documentation website does not consume Hopper tokens, we created our own. We could use Hopper Typographic tokens if needed. That would require an update to the Figma design documentation files as well as updating the whole documentation website. I think this could be a nice improvement that could be done in a separate issue / story, as it goes above the scope of this PR.